google/cloud/pubsub/samples/pubsub_samples_common.cc

```diff
@@ -19,6 +19,7 @@
 #include "google/cloud/internal/time_utils.h"
 #include "google/cloud/project.h"
 #include "absl/strings/match.h"
+#include "absl/time/time.h"
```

Comment:
Is Abseil time only used in the cc file because of the interface defined in the header?  That is, if Abseil time disappeared from the header, could it also disappear here?  If so, omit this extra include.

Reply:
Done.  Though I am guessing as to intent.